pkg/nack/nack.go

```diff
@@ -55,7 +76,12 @@ func (n *NackQueue) Push(sn uint16) {
 		n.nacks = n.nacks[:len(n.nacks)-1]
 	}
 
-	n.nacks = append(n.nacks, newNack(sn))
+	n.nacks = append(n.nacks, newNack(nackParams{
```

Comment:
Do we need params for each `nack`, can they share one params?

Reply:
Good point. Not needed. Will make a common one.

Reply:
Addressing in this commit - https://github.com/livekit/mediatransportutil/pull/15/commits/dd696167a59ffb739421d4b802ec865c7e85416b